index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+        <header class="main-header"> 
+            <a class="logo" href="#">MHabert</a>
+            <nav class="nav-container">
+                <a href="#" class="menu"><img src="menu.png"page logo=""></a>
+                
+        </nav>
+    </header>
+
+
+    <div  class="container">
+        <section class="part1">
+        <h1 class="head-name">I’m Mumbere <br>
+            Glad to see you!</h1>
+        <p class="head-para">I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+            <h2 class="sub-head">Let's Connect</h2>
+            <div class="social-media">
+                <ul>
+                    <li><a href="#"><i class=" fa fa-twitter" ></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin"  ></i></a></li>
+                    <li><a href="#"><i class="fa brand fa fa-medium" ></i></a></li>
+                    <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa brand fa fa-angellist" ></i></a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] Kindly notice the color of the social medias icons in the Figma design. It would be nice to stick to the Figma design as much as possible. 
_**HIT: You can change their color by adding this to your css file**_
```css
.social-media a {
  color: #505F79;
}
```